### DISCUSSION TITLE
can the ios app when domain 404 err will visit ip address?

### DISCUSSION THREAD

**Discussion by nenge123** [community/community#17943] [Mobile]
the gov make github.com dns ip to 127.0.0.1,
and apple ios can't change dns,then i can't using mobile app to visit

**Comment by eliperkins** [comment]
Thanks for your feedback, @nenge123! GitHub Mobile uses platform best practices, which includes using `github.com` to resolve all of our network requests.

I've jotted down your notes in our own internal issues to share your feedback with the rest of our team.
